packages/docusaurus-theme-common/src/utils/generalUtils.ts

```diff
@@ -17,3 +21,70 @@ export function useTitleFormatter(title?: string | undefined): string {
     ? `${title.trim()} ${titleDelimiter} ${siteTitle}`
     : siteTitle;
 }
+interface HasLabel {
+  label: string;
+}
+
+interface HasName {
+  name?: string;
+  imageURL?: string;
+}
+
+type Entry = HasLabel | HasName;
+
+/**
+ * Takes a list of tags or author (as provided by the content plugins),
+ * and groups them by their initials.
+ */
+function listByLetters<T extends Entry, R>(
+  items: readonly T[],
+  getLetter: (item: T) => string,
+  getLabel: (item: T) => string,
```

Comment:
Why do we need both getItemLetter and getLabel?

We'd rather get the letter from the label directly so only one is probably enough